Packs/CommonTypes/ReleaseNotes/3_6_0.md

```diff
@@ -0,0 +1,49 @@
+
+#### Indicator Fields
+
+##### New: Organization Prevalence
+
+Added new number field to indicate how frequently a specific indicator type is detected within a particular organization.
+
+##### New: Globally Prevalence
+
+Added new number field to indicate how frequently a specific indicator type is detected across all organizations.
+
+##### New: Organization First Seen
+
+Added new date field to indicate the first time a specific organization encountered an indicator.
+
+##### New: Organization Last Seen
+
+Added new date field to indicate the most recent time a specific organization encountered an indicator.
+
+##### New: Global First Seen
+
+Added new date field to indicate the first time an indicator was detected globally across all organizations.
+
+##### New: Global Last Seen
+
+Added new date field to indicate the most recent time an indicator was detected globally across all organizations.
+
+
+#### Indicator Types
+
+##### File
+
+Added default mapping to new indicator fields.
```

Comment:
Add the fields that were mapped as well.

Reply:
Added in commit db38beed6372507e3cf58090eb33520e7c27dfea